web/docs/cli.md

```diff
@@ -21,8 +21,9 @@ COMMANDS
     uninstall             Removes Wasp from your system.
   IN PROJECT
     start                 Runs Wasp app in development mode, watching for file changes.
+    start db              Starts managed development database for you.
     db <db-cmd> [args]    Executes a database command. Run 'wasp db' for more info.
-    clean                 Deletes all generated code and other cached artifacts. Wasp equivalent of 'have you tried closing and opening it again?'.
+    clean                 Deletes all generated code and other cached artifacts.                          Wasp equivalent of 'have you tried closing and opening it again?'.
```

Comment:
Does this render okay on different screen resolutions? It looks werid on my laptop 

<img width="1370" alt="Screenshot 2023-03-13 at 12 06 42" src="https://user-images.githubusercontent.com/2223680/224684469-ab862832-d5cf-4e74-af49-3b11e1f9c874.png">

Reply:
Nice catch, it was a mistake, fixed!